skimage/exposure/exposure.py

```diff
@@ -74,16 +74,18 @@ def _bincount_histogram(image, source_range):
     return hist, bin_centers
 
 
-def histogram(image, nbins=256, source_range='image', normalize=False):
+def histogram(image, nbins=256, source_range='image', normalize=False, multichannel=False):
     """Return histogram of image.
 
     Unlike `numpy.histogram`, this function returns the centers of bins and
     does not rebin integer arrays. For integer arrays, each integer value has
     its own bin, which improves speed and intensity-resolution.
 
-    The histogram is computed on the flattened image: for color images, the
-    function should be used separately on each channel to obtain a histogram
-    for each color channel.
+    If multichannel is not set, the histogram is computed on the flattened
+    image. For color or multichannel images, the function should either be
+    the function should be either be used separately on each
+    channel to obtain a histogram for each color channel with separate binning,
+    or by setting multichannel=True to use a common binning for all channels.
```

Comment:
Typos above (repeated "the function should either be"). Anyway, I think in this case we should promote the use of `multichannel` first. Suggested new wording:

```
For color or multichannel images, use the `multichannel=True` keyword argument.
This will use the same histogram binning for all channels. To obtain a different binning
for each individual channel, call the function separately for each channel, keeping
`multichannel=False`.
```

Reply:
I updated the text here to recommend `channel_axis` first (I replaced `multichannel` with `channel_axis` in this PR since we are deprecating multichannel elsewhere)